roles/common_tools/tasks/main.yml

```diff
@@ -4,15 +4,21 @@
     name: "{{ item }}"
     state: latest
   with_items:
-    - podman
-    - podman-docker
     - perl
     - git
     - make
     - libseccomp-devel
     - postfix
     - acl
 
+- name: Install basic tools for CentOS 7
+  package:
+    name: "{{ item }}"
+    state: absent
```

Comment:
Does this mean, that podman-docker will not be installed on CentOS7 machines?

Reply:
Yes, because it has a collision with docker.

`podman-docker` provides a symlink for docker. And tests are failing on CentOS 7.
E.g. https://github.com/sclorg/s2i-php-container/pull/397 and corresponding Testing Famr results: https://artifacts.dev.testing-farm.io/609a912a-2973-4b26-98be-17b5e633a6bb/

Reply:
The relevant code that is failing is here: https://github.com/sclorg/container-common-scripts/blob/master/test-lib-openshift.sh#L549

Reply:
The code is related only to OpenShift 3 use cases. AFAIK on RHEL7 we already have docker, than why not to have it also for CentOS 7

Reply:
Sure. I was just not sure about the ansible syntax.